tests/backup/backup_upgrade_test.go

```diff
@@ -595,6 +724,38 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 				dash.VerifyFatal(b.GetBackupObjectType().Type, api.BackupInfo_BackupObjectType_All, fmt.Sprintf("Verifying backup type of [%s]", b.Name))
 			}
 		})
+
+		Step("Validate the scheduled backup with partial success after upgrade and restore the same", func() {
+			log.InfoD("Verifying if the latest scheduled backup is a backup with partial success")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			latestScheduleBackupName, err := GetLatestScheduleBackupName(ctx, partialScheduledBackupName, BackupOrgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the latest schedule backup with schedule [%s]", partialScheduleName))
+			log.Infof("Validating if the latest scheduled backup [%s] is of Partial Success", latestScheduleBackupName)
+			err = BackupWithPartialSuccessCheck(latestScheduleBackupName, BackupOrgID, MaxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the backup [%s] is in partial backup state", latestScheduleBackupName))
+			log.InfoD("Restoring the scheduled backup with partial success")
+			namespaceMapping := make(map[string]string)
+			for _, namespace := range partialAppNamespaces {
+				namespaceMapping[namespace] = namespace + RandomString(4)
+			}
+			k8sCore := k8score.Instance()
+			log.FailOnError(err, "Getting the failed PVC list")
+			pvcList, err := k8sCore.GetPersistentVolumeClaims(partialAppNamespaces[0], nil)
+			for _, pvc := range pvcList.Items {
+				failedVolumes = append(failedVolumes, pvc.Name)
```

Comment:
May need to rebase the latest changes for failedVolumes

Reply:
Done